plugins_src/import_export/wpc_svg_path.erl

```diff
@@ -225,4 +758,2941 @@ getcedges([#pathop{opkind=pcurveto,x1=X1,y1=Y1,x2=X2,y2=Y2,x3=X,y3=Y}|Ops],
+substitute_commands([Op | Rest], O, CurrPos) when Op =:= "A" ->
+    {ok, NewLineTos, Rest_1, CurrPos_1} = substitute_arcs_to_lineto_abs(Rest, CurrPos),
+    substitute_commands(Rest_1, lists:reverse(NewLineTos) ++ ["L"] ++ O, CurrPos_1);
+substitute_commands([Op | Rest], O, CurrPos) when Op =:= "a" ->
+    {ok, NewLineTos, Rest_1, CurrPos_1} = substitute_arcs_to_lineto_rel(Rest, CurrPos),
+    substitute_commands(Rest_1, lists:reverse(NewLineTos) ++ ["L"] ++ O, CurrPos_1);
+
+%% Keep track of current drawing position by also keeping track of
+%% M, m, L, l, C, c  SVG path commands. The current position is needed
+%% to substitute the needed arguments of the other commands correctly.
+%%
+substitute_commands([Op,X,Y | Rest], O, CurrPos)
+    when Op =:= "M" ->
+    CurrPos_1 = CurrPos#subst_curv_pos{
+        curr_cmd = lineto_abs,
+        x_abs = subst_str_to_flt(X),
+        y_abs = subst_str_to_flt(Y)
+    },
+    substitute_commands(Rest, [Y,X,Op| O], CurrPos_1);
+substitute_commands([Op,DX,DY | Rest], O,
+    #subst_curv_pos{x_abs=XAbs,y_abs=YAbs}=CurrPos)
+    when Op =:= "m"
+->
+    CurrPos_1 = CurrPos#subst_curv_pos{
+        curr_cmd = lineto_rel,
+        x_abs = XAbs + subst_str_to_flt(DX),
+        y_abs = YAbs + subst_str_to_flt(DY)
+    },
+    substitute_commands(Rest, [DY,DX,Op| O], CurrPos_1);
+substitute_commands([Op,X,Y | Rest], O, CurrPos)
+    when Op =:= "L" ->
+    CurrPos_1 = CurrPos#subst_curv_pos{
+        curr_cmd = lineto_abs,
+        x_abs = subst_str_to_flt(X),
+        y_abs = subst_str_to_flt(Y)
+    },
+    substitute_commands(Rest, [Y,X,Op| O], CurrPos_1);
+substitute_commands([Op,DX,DY | Rest], O,
+    #subst_curv_pos{x_abs=XAbs,y_abs=YAbs}=CurrPos)
+    when Op =:= "l"
+->
+    CurrPos_1 = CurrPos#subst_curv_pos{
+        curr_cmd = lineto_rel,
+        x_abs = XAbs + subst_str_to_flt(DX),
+        y_abs = YAbs + subst_str_to_flt(DY)
+    },
+    substitute_commands(Rest, [DY,DX,Op| O], CurrPos_1);
+substitute_commands([Op,X1,Y1,X2,Y2,X,Y | Rest], O, CurrPos)
+    when Op =:= "C" ->
+    CurrPos_1 = CurrPos#subst_curv_pos{
+        curr_cmd = curveto_abs,
+        x_abs = subst_str_to_flt(X),
+        y_abs = subst_str_to_flt(Y),
+        cp2 = {subst_str_to_flt(X2), subst_str_to_flt(Y2)}
+    },
+    substitute_commands(Rest, [Y,X,Y2,X2,Y1,X1,Op| O], CurrPos_1);
+substitute_commands([Op,DX1,DY1,DX2,DY2,DX,DY | Rest], O,
+    #subst_curv_pos{x_abs=XAbs,y_abs=YAbs}=CurrPos)
+    when Op =:= "c"
+->
+    CurrPos_1 = CurrPos#subst_curv_pos{
+        curr_cmd = curveto_rel,
+        x_abs = XAbs + subst_str_to_flt(DX),
+        y_abs = YAbs + subst_str_to_flt(DY),
+        cp2 = { XAbs + subst_str_to_flt(DX2),
+                YAbs + subst_str_to_flt(DY2)}
+    },
+    substitute_commands(Rest, [DY,DX,DY2,DX2,DY1,DX1,Op| O], CurrPos_1);
+    
+substitute_commands([[NumDigit|_]=X,Y | Rest], O,
+    #subst_curv_pos{curr_cmd=lineto_abs}=CurrPos)
+    when ?IS_DIGIT(NumDigit)
+->
```

Comment:
I don't like to start the line with '->'  
'when' should be indented with 2 spaces to indicate that it belongs to the function header.

If you want I can indent the file before merging.